packages/docusaurus-plugin-content-docs/src/sidebars.ts

```diff
@@ -255,14 +256,23 @@ export const DefaultSidebars: UnprocessedSidebars = {
 };
 
 // TODO refactor: make async
-export function loadSidebars(sidebarFilePath: string): UnprocessedSidebars {
-  if (!sidebarFilePath) {
-    throw new Error(`sidebarFilePath not provided: ${sidebarFilePath}`);
+export function loadSidebars(
+  sidebarFilePath: string | false | undefined,
+): UnprocessedSidebars {
+  // In case false value, we have to set to DefaultSidebars first to generate navigation link
+  // See https://github.com/facebook/docusaurus/pull/4775
```

Comment:
not sure what you mean with that comment. Imho we just need to return an empty object if the value is false

Reply:
If we return false, so there is no previous/next button in the end of the doc. That's why I need to do so.

Reply:
but if the user use `false`, there should be no navigation, I believe this is the expected behavior. There is only a navigation if there is a sidebar